### ISSUE TITLE
Add README

### ISSUE THREAD

**Issue by htorianik** [D34DStone/gtesting_server#9]
Write some pretty README to let people know what is it all about.

**Comment by chopikus** [comment]
Yeah, what's it about, i wonder? I'm pretty interested in this project
